rust/feature-flags/src/flag_matching.rs

```diff
@@ -34,42 +179,48 @@ pub struct Person {
 // and then we can easily refactor stuff around.
 // #[derive(Debug)]
 pub struct FeatureFlagMatcher {
-    // pub flags: Vec<FeatureFlag>,
     pub distinct_id: String,
-    pub database_client: Option<Arc<dyn DatabaseClient + Send + Sync>>,
-    // TODO do I need cached_properties, or do I get them from the request?
-    // like, in python I get them from the request.  Hmm.  Let me try that.
-    // OH, or is this the FlagMatcherCache.  Yeah, so this is the flag matcher cache
-    cached_properties: Option<HashMap<String, Value>>,
+    pub team_id: TeamId,
+    pub database_client: Option<DatabaseClientArc>,
+    // TODO should all the properties by cached in the same cache?
+    // Like what's the point of having separate caches for person and group properties,
```

Comment:
I don't think we're caching person or group props anywhere yet (group type mappings are different thing - which seldom change) - person and group props can change per person and we can have millions of people, so not sure yet if it makes sense to cache these at all.